src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.exception;
+
+public class ResourceNotFoundException extends RuntimeException{
```

Comment:
would it be better to extend Exception or maybe our own ServiceException? RuntimeException is unchecked so won't forced to catch it.

Reply:
Changed to extend ServiceException